src/helpers/date-helper.ts

```diff
@@ -179,6 +179,58 @@ export const getLocaleMonth = (date: Date, locale: string) => {
     bottomValue[0],
     bottomValue[0].toLocaleUpperCase()
   );
+  switch(bottomValue){
+    case "January":
+      bottomValue = "01";
+      break;
+
+    case "February":
+      bottomValue = "02";
+      break;
+    
+    case "March":
+      bottomValue = "03";
+      break;
+        
+    case "April":
+      bottomValue = "04";
+      break;
+
+    case "May":
+      bottomValue = "05";
+      break;
+
+    case "June":
+      bottomValue = "06";
+      break;
+    
+    case "July":
+      bottomValue = "07";
+      break;
+
+    case "August":
+      bottomValue = "08";
+      break;
+    
+    case "September":
+      bottomValue = "09";
+      break;
+        
+    case "October":
+      bottomValue = "10";
+      break;
+
+    case "November":
+      bottomValue = "11";
+      break;
+
+    case "December":
+      bottomValue = "12";
+      break;
+      
+    default :
+      break; 
+  }
```

Comment:
an easier solution:
```js
const monthNames = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];

bottomValue = monthNames.indexOf(bottomValue) + 1;
bottomValue = bottomValue < 10 ? '0' + bottomValue : bottomValue;
return bottomValue.toString();

Reply:
`bottomValue` can't take `number` datatype, it can only store `string` datatype.
@rathorevaibhav

Reply:
@deepakbarmola why it cannot take number datatype? Where is that check written?

Also, we're returning a string from the function if you see my suggestion and not a number.